packages/checkout-ui-extensions/src/components/Button/Button.ts

```diff
@@ -37,6 +37,11 @@ export interface ButtonProps {
    * Whether the button should fill all available inline space.
    */
   fill?: boolean;
+  /**
+   * Specifies the inline alignment of the content.
+   * @defaultValue `center`
+   */
+  inlineAlignment?: 'leading' | 'center' | 'trailing';
```

Comment:
This type change would be an enhancement, not a breaking change as it is additive in nature.